### ISSUE TITLE
Italian postal code pattern needs to be updated

### ISSUE THREAD

**Issue by finex** [commerceguys/addressing#142]
Hi, the postal code 07029 (which refers to locality "Tempio Pausania" on "Sassari" province), is not matched by the following rule:
```
"SS": {
  "name": "Sassari",
  "iso_code": "IT-SS",
  "postal_code_pattern": "070[14]|0703[0-79]|07100"
},
```
I've also discovered that there are a lot of updates on Italian postal codes (about 250 updates/additions) but unfortunately it looks that it doesn't exists a public (free) repository of this data.

**Comment by finex** [comment]
I've found the following unofficial PDF list: http://today.it/~media/32792526955943/nuovi-cap-2.pdf
maybe it could be helpful.

**Comment by bojanz** [comment]
Can you please report this at https://github.com/google/libaddressinput/issues? Ideally we'd get this fixed upstream.

**Comment by finex** [comment]
> Can you please report this at https://github.com/google/libaddressinput/issues? Ideally we'd also get this fixed upstream.

Of course: https://github.com/google/libaddressinput/issues/199  :-)

**Comment by bojanz** [comment]
I pulled in Google's latest dataset yesterday and got this update:
https://github.com/commerceguys/addressing/commit/d54ab2a7ec1b0c08bf654889a941f1cf071587ae#diff-1c236651e792c510b546f2d39f48e489R427

Looks like that resolves it?

**Comment by bojanz** [comment]
Closing, since the fix is now in a release (v1.0.9)
